unittests/CppInterOp/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ else()
   set(EXTRA_PATH_TEST_BINARIES /CppInterOpTests/unittests/bin/$<CONFIG>/)
 endif()
 
+# Add the DispatchAPITest only when building shared libraries
+if (BUILD_SHARED_LIBS)
+  set_source_files_properties(DispatchAPITest.cpp PROPERTIES COMPILE_DEFINITIONS
+    "CPPINTEROP_LIB_DIR=\"${CMAKE_BINARY_DIR}/lib/libclangCppInterOp${CMAKE_SHARED_LIBRARY_SUFFIX}\""
```

Comment:
Please write a test and try to run that test against installed CppInterOp with the build folder deleted.